ajax/libs/parsimmon/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "parsimmon",
+  "description": "A monadic LL(infinity) parser combinator library",
+  "filename": "src/parsimmon.min.js",
+  "author": "Jeanine Adkisson <jneen at jneen dot net>",
+  "license": "MIT",
+  "repository": "https://github.com/jneen/parsimmon",
+  "keywords": [
+    "parsing",
+    "parse",
+    "parsers",
+    "parser combinators"
```

Comment:
`combinators` is enough.

Reply:
This was updated in my last commit, not sure how it didnt make it through to the pull request.

Reply:
@rickyhopkins May I know your command history?